### ISSUE TITLE
Inquiry about Discrepancies in Schaefer 200 Parcellation Labels from different toolboxes

### ISSUE THREAD

**Issue by WolkeTian** [MICA-MNI/micapipe#79]
Dear Micapipe developers,

I am writing to you as a researcher currently using your Micapipe tool for data analysis. In my study, I have utilized the Schaefer 200 parcellation scheme for network and brain region segmentation, and I have come across some discrepancies.

The Schaefer 200 parcellation that I obtained from online sources (such as the built-in version in Cat12 toolbox) appears to have different network labels compared to the ones provided in the lut_schaefer-200_mics.csv file available at https://github.com/MICA-MNI/micapipe/tree/master/parcellations/lut/lut_schaefer-200_mics.csv. For example, the last 5 labels in Cat12 are:

17Networks_RH_DefaultC_PHC_1
17Networks_RH_TempPar_1
17Networks_RH_TempPar_2
17Networks_RH_TempPar_3
17Networks_RH_TempPar_4

However, the last 5labels in lut_schaefer-200_mics.csv are:

7Networks_RH_Default_PFCdPFCm_6
7Networks_RH_Default_PFCdPFCm_7
7Networks_RH_Default_pCunPCC_1
7Networks_RH_Default_pCunPCC_2
7Networks_RH_Default_pCunPCC_3

Furthermore, I attempted to match the MNI coordinates provided in lut_schaefer-200_mics.csv to the corresponding ROI indices in the Schaefer2018_200Parcels_17Networks_order.nii file provided by Cat12. However, I found that these 200 MNI coordinates did not align with each individual ROI in the NII file. In fact, it seems that these 200 MNI coordinates correspond to 196 ROI indices (including one with an index of 0, representing an unassigned region).

I have the following questions:

Are the spatial distributions of the 200 parcellations consistent between the two tools?

If the spatial distributions are consistent, are the label orders the same or is there a difference between the 17-network and 7-network naming conventions?

If the label orders are different, how can I establish the correspondence between the two templates?

Thank you for your attention to these questions, and I am looking forward to your response.

Best regards,
Yun Tian

**Comment by WolkeTian** [comment]
From my searching, the cause of the problem seems to be that 7 and 17 have different networks indexes, ordering and affiliations. 
As discussed on the URL https://github.com/PennLINC/xcpEngine/issues/247.

**Comment by rcruces** [comment]
Did this solve your question? URL https://github.com/PennLINC/xcpEngine/issues/247.

**Comment by WolkeTian** [comment]
The discussion in this link really inspired me.
I compared the volume MNI space maps of  the Schaefer2018 17 network and the 7 network
(Schaefer2018_200Parcels_7Networks_order_FSLMNI152_2mm.nii.gz and /Schaefer2018_200Parcels_17Networks_order_FSLMNI152_2mm.nii.gz from https://github.com/ThomasYeoLab/CBIG/blob/master/stable_projects/brain_parcellation/Schaefer2018_LocalGlobal/Parcellations/MNI/), and I found that the spatial distribution is indeed completely consistent, but the index/orders of the label do not correspond. According to my calculation, the labels of the 7 networks correspond to the 1:200 normal orders of the 17 networks according to the following orders. as follows:

[2	5	7	9	11	14	1	4	6	10	12	13	22	23	24	26	27	28	29	30	15	16	17	18	44	19	20	21	31	3	8	33	37	38	34	35	25	40	41	45	47	49	50	53	52	54	46	51	48	90	55	56	57	58	59	60	32	62	63	36	42	68	69	43	70	73	64	61	93	66	67	71	39	72	82	94	97	98	99	84	86	88	74	75	76	77	81	89	91	92	95	83	65	85	87	80	96	100	78	79	103	105	106	108	111	112	104	107	109	110	113	114	124	125	126	127	128	129	130	131	132	133	134	116	117	118	119	120	121	122	135	136	115	140	141	142	137	139	143	144	145	150	151	152	153	155	156	123	157	158	165	172	174	169	154	180	159	160	191	161	162	163	164	101	167	138	146	147	173	179	186	168	184	166	175	176	170	171	181	197	177	200	178	183	196	199	192	193	194	188	185	195	190	182	198	102	187	189	148	149].

If the order of surface parcellation files is consistent with the order in MNI space, their order correspondence should be the same.
The matlab script is below:

net17 = 'Schaefer2018_200Parcels_17Networks_order_FSLMNI152_2mm.nii';
net7 = 'Schaefer2018_200Parcels_7Networks_order_FSLMNI152_2mm.nii';
% load nifti files
net17nii = niftiread(net17);
net7nii = niftiread(net7);
% reshape to 2d
net17_2d = net17nii(:);
net7_2d = net7nii(:);
% check
for i = 1:200
    % get index in 7 networks file
    indexs = find(net17_2d == i);
    net7_orders(i) = unique(net7_2d(indexs));
end
